cachi2/core/config.py

```diff
@@ -13,7 +13,7 @@ class Config(BaseModel, extra="forbid"):
 
     goproxy_url: str = "https://proxy.golang.org,direct"
     default_environment_variables: dict = {
-        "gomod": {"GOSUMDB": {"value": "off", "kind": "literal"}},
+        "gomod": {"GOSUMDB": {"value": "sum.golang.org", "kind": "literal"}},
     }
```

Comment:
Should we set GOSUMDB in the build-config at all? This would force-enable GOSUMDB in every build that uses cachi2, even if the user doesn't want that.

We definitely shouldn't force-disable it, but maybe we shouldn't set it at all and instead leave it to the user / their base image.